src/Microsoft.AspNetCore.Razor/CodeGenerators/Visitors/CSharpTagHelperFieldDeclarationVisitor.cs

```diff
@@ -168,6 +188,23 @@ private void PreAllocateUnboundTagHelperAttributes(TagHelperChunk chunk)
             }
         }
 
+        private static bool IsLiteralAttributeValue(Chunk attributeValueChunk)
```

Comment:
Don't write your own, expose what's in the `CSharpTagHelperCodeRenderer` and use that.